middleware.go

```diff
@@ -77,6 +78,12 @@ var DefaultProdStack = []rest.Middleware{
 }
 
 func SetupMiddleware(c config.ConfigReader, api *rest.Api) {
+
+	api.Use(&customheader.CustomHeaderMiddleware{
+		HeaderName:  "X-DEPLOYMENTS-VERSION",
+		HeaderValue: CreateVersionString(),
```

Comment:
Ok, but in regular builds it will also look a bit awkward. For instance master builds will return `master_a3e9adbf54fb4f606377ef134f211fe65a6224ab`. If there's a tag we'll get `0.0.2` or `integration-0.1` (these are currently defined in deployments repo). I was under the assumption that we want to have a service API level rather than service version.